### DISCUSSION TITLE
support latest node version ( 19.x ) ?

### DISCUSSION THREAD

**Discussion by zigang93** [vercel/community#1945] [Help] [answered]
unable to deploy with current version node 19 on vercel.
our team already switch over to 19.x. any advance setting able to deploy with node 19?

**Comment by swarnava** [comment, marked as answer]
Hi,

We only support even version (LTS) of Node. https://nodejs.org/en/download. The latest Node version is available is Node 18: https://vercel.com/docs/runtimes#official-runtimes/node-js/node-js-version. We will publish Node 20 support as soon as our Upstream provide make it ready.

**Reply by zigang93** [reply]
Thanks
